{"name":"doubleTestcase","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [true] but found [false]","trace":"java.lang.AssertionError: expected [true] but found [false]\n\tat org.testng.Assert.fail(Assert.java:110)\n\tat org.testng.Assert.failNotEquals(Assert.java:1413)\n\tat org.testng.Assert.assertTrue(Assert.java:56)\n\tat org.testng.Assert.assertTrue(Assert.java:66)\n\tat com.codecraft.flutter.doubleTapTest.DoubleTapTest.doubleTestcase(DoubleTapTest.java:18)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1541)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:829)\n\tat org.testng.TestRunner.run(TestRunner.java:602)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:437)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:330)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1176)\n\tat org.testng.TestNG.runSuites(TestNG.java:1099)\n\tat org.testng.TestNG.run(TestNG.java:1067)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\n"},"stage":"finished","steps":[],"attachments":[{"name":"Failure in method {0}","source":"7d1b0e50-1714-4aee-b31f-a6e217a58ddf-attachment","type":"image/png"},{"name":"doubleTestcase failed and screenshot taken!","source":"79651679-1e7c-4738-849d-204cb747318c-attachment","type":"text/plain"}],"parameters":[{"name":"environment","value":"SamsungLocal"}],"start":1683454726404,"stop":1683454732958,"uuid":"bfd78f78-4f47-4aac-af05-ad6a5a46acf0","historyId":"ac9cb4183ef9b6d04c0a6466c002a9f1","fullName":"com.codecraft.flutter.doubleTapTest.DoubleTapTest.doubleTestcase","labels":[{"name":"package","value":"com.codecraft.flutter.doubleTapTest.DoubleTapTest"},{"name":"testClass","value":"com.codecraft.flutter.doubleTapTest.DoubleTapTest"},{"name":"testMethod","value":"doubleTestcase"},{"name":"parentSuite","value":"Flutter Mobile Automation "},{"name":"suite","value":"FLUTTER-3: Regression"},{"name":"subSuite","value":"com.codecraft.flutter.doubleTapTest.DoubleTapTest"},{"name":"host","value":"Ostans-MacBook-Pro-3.local"},{"name":"thread","value":"18706@Ostans-MacBook-Pro-3.local.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[]}